### PR TITLE
Feature use redux in react

### PR DESCRIPTION
**Pull request by Graciano1997**
In this milestone I did :

- Wrapped the <App> with the <Provider> component from react-redux
- Created an initialState variable for your booksSlice,
- Display the books, received from the slice, in a reusable component
- Dispatch actions using useDispatch
- Added a <Button> component, which includes:
- An event handler that adds a book to the books array (with attributes id, title and author)
- Added a <Button> component, which includes:
- An event handler that removes a book from the books array by id
- Fix linter issues
- Update Readme

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+          <h4>{book.category}</h4>
           <h3>{book.title}</h3>
           <p>{book.author}</p>
         </div>
         <ul className={style.booOptions}>
-          <li>Comments</li>
-          <li>Remove</li>
-          <li>Edit</li>
+          <li><a href="#coment">Comments</a></li>
+          <li><a href="#remove" onClick={() => { dispatch(removeBook(book)); }}>Remove</a></li>
```
Comment (taldr27) [inline src/components/Book.jsx]: - [ ] Could you please think about moving the remove book logic to a different component? By doing this, the code will become more organized and easier to read. 

   image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello @Graciano1997, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional Readme. 
✅ Linters passing.
✅ Correct use of GitHub Flow.
✅ Good job using redux in your project.
✅ Good job adding the add and remove logic.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Graciano,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on implementing recommended changes!
🎲 ReadMe looks great!

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.jsx (hunk 1 above)
Comment (Graciano1997) [reply src/components/Book.jsx]: @taldr27  it is hard but it improves me 🙏 🙏 🙏 🙏 very much!

**Comment by Graciano1997** [comment]
@rloterh I want to thank you for recognizing all my efforts 💪 💪 🙏 🙏 .

Cheers and Happy code!

**Pull request merged**